frontend/lib/api/form/pollingstation/PollingStationFormController.tsx

```diff
@@ -137,132 +142,60 @@ export function PollingStationFormController({
-  defaultFormState = {},
+  defaultValues = undefined,
+  defaultFormState = undefined,
   defaultCurrentForm = null,
 }: PollingStationFormControllerProps) {
-  const request_path: POLLING_STATION_DATA_ENTRY_REQUEST_PATH = `/api/polling_stations/${pollingStationId}/data_entries/${entryNumber}`;
-
-  const temporaryCache = React.useRef<TemporaryCache | null>(null);
-
-  //reference to the current form on screen
-  const currentForm = React.useRef<AnyFormReference | null>(defaultCurrentForm);
-
-  //where to navigate to next
-  const [targetFormSection, setTargetFormSection] = React.useState<FormSectionID | null>(INITIAL_FORM_SECTION_ID);
-
-  // status as ref, because it needs to immediately propagate to the blocker function in `PollingStationFormNavigation`
-  const status = React.useRef<Status>("idle");
+  const request_path: POLLING_STATION_DATA_ENTRY_SAVE_REQUEST_PATH = `/api/polling_stations/${pollingStationId}/data_entries/${entryNumber}`;
+  const { client } = useApi();
 
   // TODO: #277 render custom error page instead of passing error down
   const [apiError, setApiError] = React.useState<ApiError | null>(null);
 
-  const [formState, setFormState] = React.useState<FormState>(() => {
-    const result: FormState = {
-      active: INITIAL_FORM_SECTION_ID,
-      current: INITIAL_FORM_SECTION_ID,
-      sections: {
-        recounted: {
-          index: 0,
-          id: "recounted",
-          title: "Is er herteld?",
-          isSaved: false,
-          ignoreWarnings: false,
-          errors: [],
-          warnings: [],
-        },
-        voters_votes_counts: {
-          index: 1,
-          id: "voters_votes_counts",
-          title: "Toegelaten kiezers en uitgebrachte stemmen",
-          isSaved: false,
-          ignoreWarnings: false,
-          errors: [],
-          warnings: [],
-        },
-        differences_counts: {
-          index: 2,
-          id: "differences_counts",
-          title: "Verschillen",
-          isSaved: false,
-          ignoreWarnings: false,
-          errors: [],
-          warnings: [],
-        },
-        save: {
-          index: election.political_groups.length + 3,
-          id: "save",
-          title: "Controleren en opslaan",
-          isSaved: false,
-          ignoreWarnings: false,
-          errors: [],
-          warnings: [],
-        },
-      },
-      unknown: {
-        errors: [],
-        warnings: [],
-      },
-      isCompleted: false,
-    };
+  const [values, setValues] = React.useState<PollingStationValues>();
 
-    election.political_groups.forEach((pg, n) => {
-      result.sections[`political_group_votes_${pg.number}`] = {
-        index: n + 3,
-        id: `political_group_votes_${pg.number}`,
-        title: pg.name,
-        isSaved: false,
-        ignoreWarnings: false,
-        errors: [],
-        warnings: [],
-      };
-    });
+  const initialDataRequest = useApiGetRequest<GetDataEntryResponse>(request_path);
+  useEffect(() => {
```

Comment:
```suggestion
  React.useEffect(() => {
```
And then remove the separate import on line 2

Reply:
Oops, thanks. I think that's my IDE trying to be smart.